src/unity/python/turicreate/connect/main.py

```diff
@@ -39,15 +36,18 @@ def __catch_and_log__(func, *args, **kargs):
 
 @__catch_and_log__
 def launch(server_log=None):
-    server = None
+    global __UNITY_GLOBAL_PROXY__
+    global __SERVER__
+
+    from ..extensions import _publish
```

Comment:
This should be after the set_log_progess and sever start, as importing this triggers a lot of logged code execution.